tests/test_xml.cpp

```diff
@@ -10,6 +10,12 @@ using namespace ISMRMRD;
 const std::string XML_HEADER("\
 <?xml version=\"1.0\"?>\n\
 <ismrmrdHeader xmlns=\"http://www.ismrm.org/ISMRMRD\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xs=\"http://www.w3.org/2001/XMLSchema\" xsi:schemaLocation=\"http://www.ismrm.org/ISMRMRD ismrmrd.xsd\">\n\
+	<EntityHeader>\n\
```

Comment:
This does not belong here. This relates to the transmission protocol and not the information content.